builtins/amp-img.js

```diff
@@ -200,7 +200,8 @@ export class AmpImg extends BaseElement {
       return;
     }
     // No need to generate sizes if already present.
-    const sizes = this.element.getAttribute('sizes');
+    const sizes =
+      this.element.hasAttribute('sizes') || this.img_.hasAttribute('sizes');
```

Comment:
This still generates the sizes attribute if none is specified. We need to check for the `i-amphtml-ssr` and not generate `sizes` if it is present.

Reply:
I'll add a different check for this.

Reply:
Actually, could this be a breaking change? 

There are other sources that server render images, and we don't know if they expect to have auto generated sizes.

Reply:
I've added the check above, I am still concerned it's a breaking change.